lte/gateway/c/oai/lib/hashtable/hashtable.c

```diff
@@ -457,7 +457,7 @@ hashtable_key_array_t* hashtable_ts_get_keys(hash_table_ts_t* const hashtblP) {
     return NULL;
   }
 
-  ka->keys = calloc(hashtblP->num_elements, sizeof(hash_key_t*));
+  ka->keys = calloc(hashtblP->num_elements, sizeof(hash_key_t));
```

Comment:
I think the intent here is to return the pointers to the keys? Let's first verify the usage.